### DISCUSSION TITLE
MountWizzard4 Compatiblity With The Skyx Imaging Edition

### DISCUSSION THREAD

**Discussion by gourldk** [mworion/MountWizzard4#143] [Q&A] [answered]
I have purchased a 10Micron GM2000 HPS combi and I am trying to determine what software is compatible with my new mount. I have discovered that many of the 10Micron owners are The SKYX users so went to the Software Bisque Forum and found where there was reference last to year to Mountwizzard4 but no definite answer if the models created by MW4 will work with SKYX.  Sorry for the obfuscated questions but hopefully this is the right  place to answer my questions.

**Comment by mworion** [comment, marked as answer]
@gourldk,
you are right to discuss here.
If I understand you right you are asking about mount pointing / tracking models. 
In 10micron mounts these models are calculated and managed inside the 10micron mount computer. The use (once build) is independent from the imaging software you are using. 
I know that TSX could build it's own models using TPoint, but you should decide wich way to go. To ways doing the same thing. As you bought a 10micron I would use the capabilities of the mount itself.
Does MW4 support TSX as it's imaging driver ? No. MW4 interfaces directly the device drivers (via INDI/INDOGO, ASCOM or ASCOM ALPACA).
Hope this helps.
Michel
